src/app_service/service/community/service.nim

```diff
@@ -550,6 +551,10 @@ QtObject:
             let data = CommunityChatArgs(chat: updatedChat)
             self.events.emit(SIGNAL_COMMUNITY_CHANNEL_EDITED, data)
 
+          # Handle channel members update
+          if chat.members != prev_chat.members:
```

Comment:
This will do what I want. Nim defines both [comparison operator for sequences](https://nim-lang.org/docs/system.html#%3D%3D%2Cseq%5BT%5D%2Cseq%5BT%5D) and  [comparison operator for objects](https://nim-lang.org/docs/system.html#%3D%3D%2CT%2CT_2) that does perform a "deep" comparison (every field is compared).

Also, this is exactly the same what we do in chat service already:
https://github.com/status-im/status-desktop/blob/50abc9b90f4b8653e0503e2d0d7dba099c5bdeb6/src/app_service/service/chat/service.nim#L168

Reply:
Damn, I didn't know Nim did deep comparison :exploding_head: TIL